src/pool/rpc.h

```diff
@@ -380,6 +384,17 @@ CRT_RPC_DECLARE(pool_prop_set, DAOS_ISEQ_POOL_PROP_SET, DAOS_OSEQ_POOL_PROP_SET)
 CRT_RPC_DECLARE(pool_acl_update, DAOS_ISEQ_POOL_ACL_UPDATE,
 		DAOS_OSEQ_POOL_ACL_UPDATE)
 
+#define DAOS_ISEQ_POOL_ACL_DELETE	/* input fields */	 \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
This is the usual false positive for CaRT RPC macros.